src/tools/dune

```diff
@@ -21,6 +21,8 @@
 
 (rule (with-stdout-to opam_admin_topstart.ml (echo "include Opam_admin_top\n\nlet _ = Topmain.main ()")))
 
+(rule (write-file opam_installer.mli ""))
```

Comment:
I think this could be an empty file instead. I think the less custom rules, the better

Reply:
I confess I have the opposite opinion - indeed, Dune 3.0 seems to have that, too (given that it's auto-generating these empty files in some contexts - I think including this one)

Reply:
fair enough then